### PR TITLE
centered text, added definition, changed styling for What? and Really? to differenti…

### PR DESCRIPTION
**Pull request by constantina-js**
…ate from white text

Came across an issue that you would need to click "What?" twice for the definition to be visible. Still trying to figure out how to fix it.

### CODE REVIEW

**Comment by choyiny** [comment]
#1

Looks great. Can you make a new html file under /pages/ for this? Thanks! Can you also put up a Github Pages preview?

**Comment by constantina-js** [comment]
@choyiny on it!

File: 404.html [hunk 1]
```diff
@@ -46,30 +68,54 @@
...
+    function dictionaryDefinition(){
+    	$("#curious").one('click', function(e) {
+    		$('#dictionary-definition').css("display", "block");
+    		$('#animations').css("display", "block");
+    	});
+    }
 
+    $(document).ready(function () {
+		
+		//hide dictionary-definition and Really?
```
Comment (choyiny) [inline 404.html]: use css:

```
#animations {
   display: none;
}
```

Comment (choyiny) [reply 404.html]: This may fix your issue, as you don't need to use the (document).ready function.

Comment (constantina-js) [reply 404.html]: wouldn't that affect all the other jQuery tho, because that's needed for when the page is loaded?

**Pull request closed without merging**